### PR TITLE
Add thematic & hline to displayStyles as well as more viewFlags

### PR DESCRIPTION
**Pull request by hnn0003**

### CODE REVIEW

File: packages/saved-views-client/src/models/savedViews/DisplayStyles.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 /*---------------------------------------------------------------------------------------------
-* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
-* See LICENSE.md in the project root for license terms and full copyright notice.
-*--------------------------------------------------------------------------------------------*/
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
```
Comment (roluk) [inline packages/saved-views-client/src/models/savedViews/DisplayStyles.ts:2]: Please undo all unnecessary formatting changes. In total more than 40 superfluous lines of code were added because of them. That's a large portion out of 180 lines that this PR introduced.

**Review by roluk — CHANGES_REQUESTED** [review]

File: packages/saved-views-client/src/models/savedViews/DisplayStyles.ts (hunk 1 above)
Comment (hnn0003) [reply packages/saved-views-client/src/models/savedViews/DisplayStyles.ts]: I believe I've gotten rid of all the superfluous changes now. :)

**Review by roluk — APPROVED** [review]

**Review by diegopinate — APPROVED** [review]
